src/components/Layout.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { Fragment, FunctionComponent } from 'react'
+import { SafeAreaView, ScrollView, StatusBar, View } from 'react-native'
+import { styles } from '../styles'
+
+export const Layout: FunctionComponent = ({ children }) => (
+  <Fragment>
+    <StatusBar barStyle="dark-content" />
+    <SafeAreaView style={styles.safeArea}>
+      <ScrollView
```

Comment:
The `ScrollView`might not be required sind there will not be more content to scroll down. I think this will also make the screen fixed.